systemd/turbinia_config_parser.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# # Copyright 2016 Google Inc.
+# #
+# # Licensed under the Apache License, Version 2.0 (the "License");
+# # you may not use this file except in compliance with the License.
+# # You may obtain a copy of the License at
+# #
+# #      http://www.apache.org/licenses/LICENSE-2.0
+# #
+# # Unless required by applicable law or agreed to in writing, software
+# # distributed under the License is distributed on an "AS IS" BASIS,
+# # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# # See the License for the specific language governing permissions and
+# # limitations under the License.
+
+from __future__ import unicode_literals
+import imp
+import itertools
+import os
+import sys
+
+# This Turbinia config parser is written for init scripts using systemd,
+# so this will not look for config files anywhere, except for /etc/turbinia.
+# If there's a need to run Turbinia based on the config file in a homedir,
+# you will need to run manually the start scripts.
```

Comment:
Will manually running the scripts actually work?  Won't they just fail because they can't see a config?

Reply:
Manual runs would work as long as the config file is discoverable in /etc/turbinia/.

Reply:
The comment here is "If there's a need to run Turbinia based on the config file in a homedir, you will need to run manually the start scripts.", so if the config file needs to be discoverable in /etc/turbinia, I'm not sure how running it manually will help.  Am I missing something here?

Reply:
Disregard this comment, I see you changed the wording to be more clear in the future commits.